src/org/loklak/api/admin/StatusService.java

```diff
@@ -114,6 +115,48 @@ public JSONObject serviceImpl(Query post, HttpServletResponse response, Authoriz
         system.put("load_process_cpu", OS.getProcessCpuLoad());
         system.put("server_threads", LoklakServer.getServerThreads());
         system.put("server_uri", LoklakServer.getServerURI());
+        system.put("Xmx", xmx);
+
+        JSONObject searchCount = new JSONObject(true);
+        String search_count_low = DAO.getConfigData("search.count.low", "");
+        String search_count_default = DAO.getConfigData("search.count.default", "");
+        String search_count_max_public = DAO.getConfigData("search.count.max.public", "");
+        String search_count_max_localhost = DAO.getConfigData("search.count.max.localhost", "");
+        String search_timeout = DAO.getConfigData("search.timeout", "");
+        searchCount.put("search_count_low", search_count_low);
+        searchCount.put("search_count_default", search_count_default);
+        searchCount.put("search_count_max_public", search_count_max_public);
+        searchCount.put("search_count_max_localhost", search_count_max_localhost);
+        searchCount.put("search_timeout", search_timeout);
```

Comment:
hey, could you create different methods for different blocks of code. This will separate the `how-it-works` code from `what-is-to-be-outputted` code